operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 version: 3.1.0-weekly.203
```

Comment:
Should we bump the weekly here or update this to reflect the Mimir tags?

Reply:
let's do in separate step